google_compute_engine/accounts/oslogin_utils.py

```diff
@@ -107,17 +113,24 @@ def UpdateOsLogin(self, enable, two_factor=False):
     Returns:
       int, the return code from updating OS Login, or None if not present.
     """
-    status = self._GetStatus()
-    if status is None:
+    oslogin_status = self._GetStatus(two_factor=False)
+    two_factor_status = self._GetStatus(two_factor=True)
```

Comment:
you don't need to do two checks, following #700 you can simple add two_factor=True if needed and get a true/false response

Reply:
If OS Login Two Factor is set to false in metadata, we still need to know whether OS Login and OS Login two factor were enabled previously. There's multiple similar cases. We end up needing to know both states.

Reply:
Updated the logic. Thanks for the feedback.